hotshot-events-service/Cargo.toml

```
@@ -16,6 +16,7 @@ hotshot-types = { workspace = true }
 rand = { workspace = true }
 serde = { workspace = true }
 snafu = "0.8"
+primitive-types.workspace = true
```

`cargo sort` doesn't like this

```
called `Result::unwrap()` on an `Err` value: TomlError { message: "TOML parse error at line 19, column 16\n   |\n19 | primitive-types.workspace = true\n   |                ^\nUnexpected `.`\nExpected `=`\n" }
```
```suggestion
primitive-types = { workspace = true }
```